docs(profile readme): bring more love in Tractus-X (#31)
eclipse-tractusx/.github: profile/README.md

# Welcome to Eclipse Tractus-X  🙌

![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)

## About Eclipse Tractus-X

> Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
> collaboration across the supply chain using trusted open standards.

Reviewer: ```suggestion
Eclipse Tractus-X is a collaborative, open-source project aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless collaboration across the supply chain using trusted open standards.
```

> Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.

Reviewer: ```suggestion
Eclipse Tractus-X is the home of the reference implementations used in [Catena-X](https://catena-x.net). The project develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```

> This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.

Reviewer: ```suggestion
This includes reference implementations and models for **supply chain**, **compliance**, **sustainability** and **more**.
```

For the latest Tractus-X release, see the [Changelog](https://eclipse-tractusx.github.io/CHANGELOG) on our [webpage](https://eclipse-tractusx.github.io/).

## Our Vision

> Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive and many other industries that:
> 
> - Enables collaboration between suppliers, manufacturers, and partners.
> - Facilitates transparency and sustainability in automotive industrial processes.
> - Drives innovation across industries through open and community-driven standards.
> - Ensures data sovereignty and privacy for all participants.
> - Offers an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.

Reviewer: ```suggestion
Our goal is to create open, scalable and secure reference implementations for a digital ecosystem in the automotive and many other industries that:

- Enable collaboration between suppliers, manufacturers, and partners.
- Facilitate transparency and sustainability in automotive industrial processes.
- Drive innovation across industries through open and community-driven standards.
- Ensure data sovereignty and privacy for all participants.
- Offer an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.
```

## Related Projects

- **[Eclipse Dataspace Components](https://projects.eclipse.org/projects/technology.edc)**: A solution for secure and standardized data sharing between partners.
- **[Catena-X Ecosystem](https://catena-x.net/en/)** operated for example by **[Cofinity-X](https://www.cofinity-x.com/)**

## How to Get Involved

We are an open-source community, and we welcome contributors from all backgrounds. You can get involved in many ways:

- **Join our discussions**: Check out the issues in our repositories to see current challenges and join the conversation.
- **Contribute**: Whether it's code, documentation, or ideas, contributions are always welcome! Start by reading our [Contributing Guide](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute).
- **Attend our events**: Participate in our [regular](https://eclipse-tractusx.github.io/community/open-meetings#regular-meetings) and [one-time](https://eclipse-tractusx.github.io/community/open-meetings#one-time-meetings).
- **Collaborate**: Partner with us to help shape the future of digital transformation in the automotive and manufacturing industries.

> - **Repositories**: Check out our [GitHub repositories](https://github.com/eclipse-tractusx) to see our latest projects and contributions.

Reviewer: ```suggestion
- **Repositories**: Check out our [GitHub repositories](https://github.com/orgs/eclipse-tractusx/repositories) to see our latest projects and contributions.
```

Reviewer 2: nice point

- **Security**: Please report any security issues to **TBD** following our [TRG 8 - Security](https://eclipse-tractusx.github.io/docs/release/trg-8/trg-8-01) and [sig-security](https://github.com/eclipse-tractusx/sig-security).

## Useful Links

- [Tractus-X Official Website](https://www.tractus-x.com)
- [Getting Started](https://eclipse-tractusx.github.io/docs/getting-started)
- [Eclipse Foundation](https://www.eclipse.org)
- [Eclipse Foundation Project handbook](https://www.eclipse.org/projects/handbook/)
- [Catena-X Automotive Network e.V.](https://catena-x.net)
- [Catena-X - Library](https://catenax-ev.github.io/)
- [Eclipse Tractus-x Project Page](https://projects.eclipse.org/projects/automotive.tractusx)

## Stay Connected

- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)

## Our Community

> Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our project leads of Eclipse Tractus-X:

Reviewer: ```suggestion
Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) and our [project leads](https://projects.eclipse.org/projects/automotive.tractusx/who) of Eclipse Tractus-X:
```

> | ![Stephan Bauer](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Mathias Brunkow Moser](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |

Reviewer: I wouldn't single out the project leads here so much, as our community is driven by committers, contributors and project leads. In regards to community, I'd find links to our Matrix chat and dev mailing list more meaningful.

Reviewer 2: Its cool so see some faces in the main readme file. If I am honest I would let us there so that people can contact us.

Author: @evegufy, @matbmoser  I updated the description and added the links to mailinglist and matrix chat :) -> The project lead faces are still there :) Do you really want to delete them? :)

Reviewer: @stephanbcbauer I'd really not treat project leads any different, so, yes, I'd go for deleting them, sorry if that's not the anticipated answer :)

Author: OK :) will delete us/them :)

# Welcome to Eclipse Tractus-X  🙌

![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)

## About Eclipse Tractus-X

Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
collaboration across the supply chain using trusted open standards.

> Tractus-X is the home of the refernce implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.

Reviewer 3: ```suggestion
Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
```

Reviewer 3: Should Catena-X be explicitly mentioned here? Or perhaps formulate it more openly so not to restrict it to Catena-X as Catena-X is (potentially) just the first, but other initiatives may join?

Reviewer 4: Should be mentioned - transparency is key - no need to hide - we just need to keep it open and inviting for others and i think Stephans current wording reflects that.

This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.

For the latest Tractus-X release, see the [Changelog](https://eclipse-tractusx.github.io/CHANGELOG) on our [webpage](https://eclipse-tractusx.github.io/).

## Our Vision

> Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive industry that:

Reviewer 4: That however is too much catena-x I'd say - tractus-x does not create the ecosystem but supports it by providing a development environment and KITs.

Author: It's never too much Catena-X ;)

Author: Is this comment related to the bullet points? Or the first sentence in the Vision?

Reviewer 4: The first sentence in the vision

- Enables collaboration between suppliers, manufacturers, and partners.
- Facilitates transparency and sustainability in automotive industrial processes.
- Drives innovation across industries through open and community-driven standards.
- Can easily be adapted/extended to other industries.
- Ensures data sovereignty and privacy for all participants.
- Offers an easy entry to various use cases through our [KITs (Keep It Together)](https://eclipse-tractusx.github.io/Kits) approach.

> ## Key Projects

Reviewer 4: Related projects instead of key projects? Tractus-X is a project itself...

Author: Adapted the headline and also added suggestion for Catena-X Ecosystem and the operating company. .... EDC still OK? What else?

- **[Eclipse Dataspace Components](https://projects.eclipse.org/projects/technology.edc)**: A solution for secure and standardized data sharing between partners.

> - What else?

Reviewer 4: Deleted before merging

Author: Yes, is just a reminder ;)

Author: done

> - What else?

Reviewer 4: I'd list Catena-X here

Author: Can you please check the wording and links again?

## How to Get Involved

We are an open-source community, and we welcome contributors from all backgrounds. You can get involved in many ways:
- **Join our discussions**: Check out the issues in our repositories to see current challenges and join the conversation.
- **Contribute**: Whether it's code, documentation, or ideas, contributions are always welcome! Start by reading our [Contributing Guide](https://eclipse-tractusx.github.io/docs/oss/how-to-contribute).
- **Attend our events**: Participate in our [regular](https://eclipse-tractusx.github.io/community/open-meetings#regular-meetings) and [one-time](https://eclipse-tractusx.github.io/community/open-meetings#one-time-meetings).

> - **Collaborate**: Partner with us to help shape the future of digital transformation in the automotive and manufacturing industries.

Reviewer 4: Can we leave a contact address of a project lead?

Author: @eclipse-tractusx/automotive-tractusx-project-leads what do you think?

# Welcome to Eclipse Tractus-X  🙌

![Tractus-X Logo](https://avatars.githubusercontent.com/u/102587798?s=200&v=4)

## About Eclipse Tractus-X

Eclipse Tractus-X is a collaborative, open-source initiative aimed at driving the digital transformation of the automotive industry and beyond. Our mission is to enable secure, efficient data exchange and seamless
collaboration across the supply chain using trusted open standards.

Tractus-X is the home of the reference implementations used in Catena-X. Tractus-X develops solutions that foster interoperability between various stakeholders, providing an open ecosystem for data-driven applications.
This includes reference implementations and models for **supply chain**, **compliance**, **sustainability**, and **more**.

For the latest Tractus-X release, see the [Changelog](https://eclipse-tractusx.github.io/CHANGELOG) on our [webpage](https://eclipse-tractusx.github.io/).

## Our Vision

> Our goal is to create an open, scalable, and secure digital ecosystem (refernce implementations) for the automotive industry that:

Reviewer 2: For the automotive and many other industries?

Author: done

- Enables collaboration between suppliers, manufacturers, and partners.
- Facilitates transparency and sustainability in automotive industrial processes.
- Drives innovation across industries through open and community-driven standards.

> - Can easily be adapted/extended to other industries.

Reviewer 2: Maybe this is already mentioned here

Author: yes already mentioned in the vision, first sentence... @HFocken fine with the first sentence now?

- **Security**: Please report any security issues to **TBD** following our [TRG 8 - Security](https://eclipse-tractusx.github.io/docs/release/trg-8/trg-8-01) and [sig-security](https://github.com/eclipse-tractusx/sig-security).

## Useful Links

- [Tractus-X Official Website](https://www.tractus-x.com)
- [Getting Started](https://eclipse-tractusx.github.io/docs/getting-started)
- [Eclipse Foundation](https://www.eclipse.org)
- [Eclipse Foundation Project handbook](https://www.eclipse.org/projects/handbook/)
- [Catena-X Automotive Network e.V.](https://catena-x.net)
- [Catena-X - Library](https://catenax-ev.github.io/)
- [Eclipse Tractus-x Project Page](https://projects.eclipse.org/projects/automotive.tractusx)

## Stay Connected

- Join our mailing list: [subscribe here](https://accounts.eclipse.org/mailing-list/tractusx-dev)
- Follow us on our [webpage](https://eclipse-tractusx.github.io/) and our [news sectcion](https://eclipse-tractusx.github.io/blog)
- Join our matrix chats at: [Tractus-X](https://matrix.to/#/#automotive.tractusx:matrix.eclipse.org)

## Our Community

Meet the project leads of Eclipse Tractus-X:

> | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/stephanbcbauer?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/matbmoser?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Evelyn Gurschler](https://images.weserv.nl/?url=avatars.githubusercontent.com/evegufy?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Daniel Miehle](https://images.weserv.nl/?url=avatars.githubusercontent.com/danielmiehle?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d) | ![Björn Roy](https://images.weserv.nl/?url=avatars.githubusercontent.com/giterrific?v=4&h=50&w=50&fit=cover&mask=circle&maxage=7d)  |

Reviewer 2: Evelyn is mentioned three times in the "alt" option. Maybe we should add our names when specified

Author: copy pate :) haha

Author: done

|                                     **[Stephan Bauer](https://github.com/stephanbcbauer)**                                      | **[Mathias Brunkow Moser](https://github.com/matbmoser)**           |                                                 **[Evelyn Gurschler](https://github.com/evegufy)**                                                 |                                                **[Daniel Miehle](https://github.com/danielmiehle)**                                                 |                                               **[Björn Roy](https://github.com/giterrific)**                                               |

> Meet the [contributors](https://projects.eclipse.org/projects/automotive.tractusx/who) and [committer](https://projects.eclipse.org/projects/automotive.tractusx/who) of Eclipse Tractus-X.

Reviewer 2: Should not the committers come first?

Author: You mean before the contributors? Or Contributor and Committer before Project lead?

Author: done
